eudemo/eudemo/radial_build.py

```diff
@@ -218,7 +218,7 @@
         "alstroh": 6.6e8,
         "psepbqarmax": 9.2,
         "aspect": 3.1,
-        "m_s_limit": 0.1,
+        # "m_s_limit": 0.1,
```

Comment:
for my own info, why is this commented?

Reply:
Sorry, this was just to test that a warning came up, will put it back